big_tests/test.config

```diff
@@ -221,7 +221,9 @@
        {redis, global, default, [{workers, 10}, {strategy, random_worker}], []},
        {rdbms, global, default, [{workers, 5}],
         [{server, {mysql, \"localhost\", \"ejabberd\", \"ejabberd\", \"mongooseim_secret\",
-                   [{verify, verify_peer}, {cacertfile, \"priv/ssl/cacert.pem\"}]}}]}
+                   [{verify, verify_peer}, {cacertfile, \"priv/ssl/cacert.pem\"},
+                    %% To avoid ssl-handshake error in erlang 20 and older
```

Comment:
And newer I think. This change affects Erlang 20, 21 and probably future releases.